recipes/snappy/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: snappy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/google/snappy.git
```

Comment:
Also, they may have run autoconf and other things so it is simpler to build. With many google libraries the standard has been downloading gtest and gmock, though they may have tarballs where this is already done.